.claude/skills/interview-parameters/probe.py

```
@@ -0,0 +1,353 @@
+    # every other one. At 11 parameters that is minutes; at 43 it is an overnight
+    # job, and nothing used to say so until you were already inside it.
+    worst_masking = sum(
+        len(sweep_values(p)) * max(len(targets) - 1, 0) for p in targets
+    )
+    worst_hours = per_run * worst_masking / 3600
+    print(f"  one run takes {per_run:.1f}s")
+    print(f"  {planned} sweep runs planned  ->  about "
+          f"{per_run * planned / 60:.1f} min")
+    print(f"  masking re-probe is O(n^2): worst case {worst_masking} more runs "
+          f"-> up to {worst_hours:.1f} hours")
+    print(f"  raise --budget to trade accuracy for time\n", flush=True)
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)